test/Dialect/SV/hw-legalize-modules-packed-arrays.mlir

```diff
@@ -4,15 +4,13 @@ module attributes {circt.loweringOptions = "disallowPackedArrays"} {
 hw.module @reject_arrays(in %arg0: i8, in %arg1: i8, in %arg2: i8,
                          in %arg3: i8, in %sel: i2, in %clock: i1,
                          out a: !hw.array<4xi8>) {
-  // This needs full-on "legalize types" for the HW dialect.
-  
   %reg = sv.reg  : !hw.inout<array<4xi8>>
   sv.alwaysff(posedge %clock)  {
-    // expected-error @+1 {{unsupported packed array expression}}
     %0 = hw.array_create %arg0, %arg1, %arg2, %arg3 : i8
     sv.passign %reg, %0 : !hw.array<4xi8>
   }
 
```

Comment:
Could you add `CHECK` lines for this change?

Reply:
I could, but this particular test case still fails, due to lack of handling of module input/outputs, so I'm concerned that will just make this more fragile. I changed the file 1. to remove the first `expected-error` which no longer fires (the second one still does), and to remove the "This needs legalize types" comment, which seems a bit misleading after this patch.

Reply:
Ah, understood. That makes sense to me!